packages/go/graphschema/ad/ad.go

```diff
@@ -91,6 +91,7 @@ var (
 	TrustedForNTAuth                = graph.StringKind("TrustedForNTAuth")
 	EnterpriseCAFor                 = graph.StringKind("EnterpriseCAFor")
 	CanAbuseUPNCertMapping          = graph.StringKind("CanAbuseUPNCertMapping")
+	CanAbuseWeakCertBinding         = graph.StringKind("CanAbuseWeakCertBinding")
```

Comment:
This change should originate from ad.cue since this file is generated from the contents of the cue schema.